web/import.data.php

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
```

Comment:
Not following the goal of this page -- from above it looks like you've switched entirely to Shiny.  Should this file be dropped entirely?

Reply:
Yes this file should be dropped. I was unaware that it was in the PR. My apologies.